src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -878,6 +879,17 @@ public function providePathsForMakePathRelative()
             array('C:/aa/bb/../../cc', 'C:/aa/../dd/..', 'cc/'),
             array('C:/../aa/bb/cc', 'C:/aa/dd/..', 'bb/cc/'),
             array('C:/../../aa/../bb/cc', 'C:/aa/dd/..', '../bb/cc/'),
+            array('aa/bb', 'aa/cc', '../bb/'),
```

Comment:
Yes, it’s line 1102 in the current master branch:

https://github.com/symfony/symfony/blob/8088661d6be2a83908b07c5bc1bd797f47707d72/src/Symfony/Component/Filesystem/Tests/FilesystemTest.php#L1102